src/Components/Common/PageTitle.tsx

```diff
@@ -3,8 +3,10 @@ import Breadcrumbs from "./Breadcrumbs";
 import PageHeadTitle from "./PageHeadTitle";
 import { classNames } from "../../Utils/utils";
 import useAppHistory from "../../Common/hooks/useAppHistory";
+import { ReactNode } from "react";
 
 export interface PageTitleProps {
+  children?: ReactNode;
```

Comment:
In https://github.com/coronasafe/care_fe/pull/5838#discussion_r1252839842 I've requested changes to use the `Page` component instead of adding children prop to the `PageTitle` component.

Reply:
@rithviknishad  i have removed children props from PageTitle and used Page component as wrapper everywhere